flaskapp/view_functions/ivrflow.py

```diff
@@ -29,23 +29,25 @@ def voice_joined():
     else:
         resp.say(f'We got your answer {answer}. We hope you will back us later. Take care.')
         resp.hangup()
-    return(str(resp))
+    return (str(resp))
+
 
 def voice():
     """ Function for answering from any call to Main Number of the IVR """
     resp = VoiceResponse()
     tel = request.values['From']
     user = check_new_user(tel)
-    if user == 'Exist':resp.dial(optional_number)
+    if user == 'Exist':
```

Comment:
Here, I would modify `check_new_user`  to return boolean value, and renamed it (to `is_user_new`); e.g.  `is_user_new(tel)` should return `True` or `False` instead of `Exist` string (as `check_new_user` does); Even from performance point of view, I almost sure, that strings comparison is a bit slower than just follow `True` or `False` conditional branches.  
Current changes is ok; they made code a bit pythonic; one-line constructions like `if a == b: do_something()` are discouraged by pep8.

Reply:
done